calico_containers/integrations/kubernetes/calico_kubernetes.py

```diff
@@ -319,23 +333,113 @@ def _apply_tags(self, profile_name, pod):
         :return:
         """
         print('Applying tags')
-        try:
-            labels = pod['metadata']['labels']
-        except KeyError:
-            # If there are no labels, there's no more work to do.
-            print('No labels found in pod %s' % pod)
-            return
-
-        for k, v in labels.iteritems():
-            tag = '%s_%s' % (k, v)
-            tag = tag.replace('/', '_')
-            print('Adding tag ' + tag)
+
+        # Grab namespace and create a tag if it exists.
+        NAMESPACE_PREFIX = self._get_metadata(pod, 'namespace')
```

Comment:
Not quite what I meant when I said you should use NAMESPACE_PREFIX.

I was thinking of the 'namespace' string as a constant which indicates a namespace.

e.g.
```
tag = self._label_to_tag(NAMESPACE_PREFIX, namespace, None)
```

Looking at this closer, I'm not sure that self._label_to_tag is the correct method to be using to generate the namespace tag.  It works, since it is similar function, but the abstraction seems wrong, since we're not actually creating a tag from a label, but instead from a namespace and a defined prefix.


Maybe something like this?
```
....
ns_tag = _escape_string("%s=%s" % (NAMESPACE_PREFIX, namespace))
....

def _label_to_tag(self, label_key, label_value, namespace):
    ....
    tag = "%s/%s=%s" % (namespace, label_key, label_value)
    return self._escape_string(tag)

def _escape_string(self, string):
    """ Escapes the given string for use in Calcio profiles / tags """
   ....

```